### PR TITLE
Update example and dataflow-example curricula with simulator tile and variable related buttons.

### PR DESCRIPTION
**Pull request by tealefristoe**

### CODE REVIEW

File: curriculum/dfe/content.json [hunk 1]
```diff
@@ -6,7 +6,11 @@
   "config": {
     "placeholderText": "This is where users can type text",
     "defaultDocumentType": "personal",
-    "settings": {"table": {"numFormat": ".2~f"}, "text": {"tools": ["bold", "subscript", "superscript"]}},
```
Comment (lbondaryk) [inline curriculum/dfe/content.json:9]: "settings" seems to be duplicated which I don't think works?

File: curriculum/dfe/content.json [hunk 2]
```diff
@@ -24,6 +28,16 @@
       {"id": "Table", "title": "Table", "isTileTool": true},
       {"id": "Drawing", "title": "Drawing", "isTileTool": true},
       {"id": "Dataflow", "title": "Dataflow", "isTileTool": true},
+      {
+        "id": "Simulator",
+        "title": "Simulator",
+        "isTileTool": true
+      },
```
Comment (lbondaryk) [inline curriculum/dfe/content.json:35]: Did you want to start this way with the Simulator on the toolbar? That is fine if so, just so we realize the ultimate goal is to show in the curriculum but not in the toolbar.

File: curriculum/dfe/content.json (hunk 1 above)
Comment (tealefristoe) [reply curriculum/dfe/content.json:9]: I'm only seeing one copy of "settings" in there. What am I missing?

File: curriculum/dfe/content.json (hunk 2 above)
Comment (tealefristoe) [reply curriculum/dfe/content.json:35]: Yeah I think it's good to have a button that works, even if we don't use it in any live units. I've got a copy on the left in this curriculum so you can add a simulator tile in either way 👍

File: curriculum/dfe/content.json (hunk 1 above)
Comment (lbondaryk) [reply curriculum/dfe/content.json:9]: ok, now it's gone. I swear to you there was one but I don't see it now so nevermind.

**Review by lbondaryk — APPROVED** [review]

**Pull request merged**